docker-compose.yaml

```diff
@@ -24,7 +24,13 @@ services:
       - SERVERFULL_RUNTIME_STATS_DATADOG_ADDRESS=statsd:8126
       - SERVERFULL_RUNTIME_SIGNALS_INSTALLED=OS
       - SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS=15 2
-      - NEXPOSE_SITE_ASSET_PAGE_SIZE=100
+      ## Nexpose and producer configuration can be set here. Sensitive information
```

Comment:
suggestion: to help illustrate this, you don't _have_ to provide a value for these variables: https://docs.docker.com/compose/environment-variables/

Reply:
I didn't know that, but I would absolutely prefer to not have to set values for these. Thanks!